APMrover2/Parameters.h

```diff
@@ -36,6 +36,7 @@ class Parameters {
         k_param_pivot_turn_angle,
         k_param_rc_13_old,  // unused
         k_param_rc_14_old,  // unused
+        k_param_pivot_turn_rate,
```

Comment:
I don't think this is good as k_param_pivot_turn_rate is 20 and 20 is already taken by k_param_rssi_pin 
Better put the parameter in g2

Reply:
nice catch @khancyr!

Reply:
fixed, Thanks!